smart-home-editor-extension/src/browser/smart-home-editor-frontend-module.ts

```diff
@@ -25,41 +25,55 @@ import '../../src/browser/style/index.css';
 
 const LIGHT_THEME_ID = "light"
 
-class MyResourceSaveable extends ResourceSaveable {
-  constructor(resource: Resource, getData: () => any, private messageService: MessageService) {
-    super(resource, getData);
+class MyResourceSaveable extends DirtyResourceSavable {
+  constructor(resource: Resource, getData: () => any, widgetManager: WidgetManager, messageService: MessageService) {
+    super(resource, getData, widgetManager, messageService);
   }
   onSave(data: any) {
-    return postRequest(window.location.protocol + '//' + window.location.hostname + ':9091/services/convert/json',
-      JSON.stringify(data),
-      'application/json')
-      .then(response => response.text(), () => this.messageService.error('Save was not possible.'))
+    return super.onSave(data);
+    // return postRequest(window.location.protocol + '//' + window.location.hostname + ':9091/services/convert/json',
```

Comment:
I think this should not be commented out. We still need this logic. Did you maybe comment it out for testing purposes and forgot to comment it in again?